### ISSUE TITLE
Always include alt attribute for img tags

### ISSUE THREAD

**Issue by alexander-dubinski** [JD-Herrera/ColmarAcademy#3]
https://github.com/JD-Herrera/ColmarAcademy/blob/ae904a11248339c04ccf10d8e416c11503be4ae9/index.html#L97
An alt attribute on the img tags are essential for when the image cannot be loaded and so screen readers know what the image contains. Always use them on img tags
